packages/web/src/components/track/mobile/ConnectedTrackTile.tsx

```diff
@@ -146,6 +146,11 @@ const ConnectedTrackTile = ({
   const { artist_pick_track_id, user_id, handle, name, is_verified } =
     getUserWithFallback(user)
 
+  // Check for comment count in tanquery cache first because that value gets polled and updated more often
+  const comment_count =
+    useCachedTrackCommentCount(track_id, currentUserId)?.data?.currentValue ??
```

Comment:
this does make me a little nervous but i suppose it's not a huge perf it as long as usCacheTrackCommentCount not firing a lot

Reply:
I ended up changing this. For a few different reasons I decided to not reference tan query cache anywhere but comments and instead just keep dispatching events to the legacy cache to update it.